src/lib.rs

```diff
@@ -1499,7 +1499,7 @@ impl Connection {
         }
 
         // Calculate available space in the packet based on congestion window.
-        let mut left = cmp::min(self.recovery.cwnd(), b.cap());
+        let mut left: u64 = cmp::min(self.recovery.cwnd(), b.cap()) as u64;
```

Comment:
IMO `left` should be kept as `usize` since it tracks the size of a memory buffer.

Reply:
Updated at 1116ab8